### PR TITLE
Add recaptcha v3 support and cleanup

### PR DESCRIPTION
**Pull request by blushi**

### CODE REVIEW

File: server/Server.ts [hunk 1]
```diff
@@ -41,9 +41,6 @@ const corsOptions = (req, callback) => {
   callback(null, options) // callback expects two parameters: error and options
 }
 
-const redisClient = redis.createClient(process.env.REDIS_URL);
-console.log('Connecting to Redis at: ', process.env.REDIS_URL)
```
Comment (blushi) [inline server/Server.ts:45]: @mhagel was this really needed given that we're already instantiating a new `Keyv` in the image processing middleware?

Comment (mhagel) [reply server/Server.ts:45]: Hmm, I _thought_ it was necessary to instantiate the client here before the `Keyv` namespace, but I may be wrong. This seems to still be working.

File: server/routes/recaptcha.ts [hunk 2]
```diff
@@ -22,7 +23,14 @@ router.post('/recaptcha/verify', bodyParser.json(), async (request, response: ex
       console.error(error)
       response.status(500).send(error);
     })
+}
 
+router.post('/recaptcha/verify', bodyParser.json(), (request, response: express.Response) => {
```
Comment (mhagel) [inline server/routes/recaptcha.ts:28]: 👍

**Review by haveanicedavid — APPROVED** [review]
Looks great to me 😎

**Pull request merged**